assets/chef-recipes/cookbooks/mdbe_build/recipes/default.rb

```diff
@@ -0,0 +1,406 @@
+    execute 'install development tools' do
+      command "sudo yum -y groupinstall 'Development Tools'"
+    end
+    package 'ccache'
+    package 'subversion'
+    package 'python-devel'
+    package 'python-pip'
+    package 'libasan'
+    package 'clang'
+    package 'checkpolicy'
+    package 'policycoreutils-python'
+    package 'mhash-devel'
+    package 'gnutls-devel'
+    package 'scons'
+    package 'systemd-devel'
+    package 'cracklib-devel'
+    package 'Judy-devel'
+    package 'patch'
+    package 'perl-Test-Base'
+    package 'jemalloc'
+  when 8 # CentOS 8
+    package 'cmake'
+    package 'checkpolicy'
+    package 'bison'
+    package 'lz4-devel'
+    package 'kernel-headers'
+    package 'libasan'
+    package 'policycoreutils'
+    package 'gnutls-devel'
+    package 'Judy'
+    package 'systemd-devel'
+    package 'cracklib'
+    package 'patch'
+    package 'redhat-lsb-core'
+    package 'patch'
+    package 'perl-Memoize.noarch'
+    package 'perl-Getopt-Long'
+    package 'jemalloc'
+    execute 'install epel-release' do
+      command 'sudo dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm'
+    end
+    execute 'install development tools' do
+      command "sudo dnf -y groupinstall 'Development Tools'"
+    end
+  end
+when 'opensuseleap' # Suse 15
+  package 'gcc-c++'
+  package 'cmake'
+  package 'libaio-devel'
+  package 'pam-devel'
+  package 'wget'
+  package 'libgnutls-devel'
+  package 'bison'
+  package 'ncurses-devel'
+  package 'libxml2-devel'
+  package 'libcurl-devel'
+  package 'rsync'
+  package 'socat'
+  package 'lsof'
+  package 'tar'
+  package 'gzip'
+  package 'bzip2'
+  package 'rpm-build'
+  package 'checkpolicy'
+  package 'policycoreutils'
+  package 'curl'
+  package 'perl'
+  package 'valgrind-devel'
+  package 'sudo'
+  package 'git'
+  package 'scons'
+  package 'perl-XML-Simple'
+  package 'systemd-devel'
+  package 'check-devel'
+  package 'snappy-devel'
+  package 'expect'
+  package 'jemalloc'
+  package 'net-tools'
+  package 'flex'
+  package 'libboost_*-devel'
+  package 'autoconf'
+  package 'automake'
+  package 'libtool'
+when 'suse' # Sles 12
+  package 'gcc-c++'
+  package 'libaio-devel'
+  package 'pam-devel'
+  package 'perl-XML-Simple'
+  package 'libgnutls-devel'
+  package 'bison'
+  package 'systemd-devel'
+  package 'ncurses-devel'
+  package 'libxml2-devel'
+  package 'libcurl-devel'
+  package 'rsync'
+  package 'socat'
+  package 'lsof'
+  package 'tar'
+  package 'gzip'
+  package 'bzip2'
+  package 'rpm-build'
+  package 'checkpolicy'
+  package 'policycoreutils'
+  package 'curl'
+  package 'perl'
+  package 'check-devel'
+  package 'valgrind-devel'
+  package 'wget'
+  package 'sudo'
+  package 'git'
+  package 'scons'
+  package 'boost-devel'
+  package 'snappy-devel'
+  package 'expect'
+  package 'net-tools'
+  package 'flex'
+  package 'autoconf'
+  package 'automake'
+  package 'libtool'
+end
+
+execute 'install cmake' do
```

Comment:
This will be executed for all the platforms. Why do we even bother to install the cmake from the repositories?